i18n/en.json

```diff
@@ -212,9 +211,20 @@
   "admin.complianceExport.exportJobStartTime.description": "Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.",
   "admin.complianceExport.exportJobStartTime.example": "E.g.: \"02:00\"",
   "admin.complianceExport.exportJobStartTime.title": "Compliance Export time:",
+  "admin.complianceExport.globalRelaySettings.title": "GlobalRelay Settings:",
```

Comment:
Propose we remove this. It's inconsistent with our other pages

![image](https://user-images.githubusercontent.com/13119842/37266887-89a00bdc-2593-11e8-948f-6a43b42bb394.png)